cves/CVE-2012-0858.yml

```diff
@@ -260,10 +259,8 @@ interesting_commits:
       * A change that looks really sketchy
       * Commits where a developer mentions security in their message or comments
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: 02591641f88097aec2a573f0ae384c8b87bcfe3b
```

Comment:
It's crazy how the dev team fixed a buffer overflow but didn't realize the invalid free!